docs/nextConfigDocsInfra.js

```diff
@@ -71,6 +71,8 @@ function withDocsInfra(nextConfig) {
       NETLIFY_DEPLOY_URL: process.env.DEPLOY_URL,
       // Name of the site, its Netlify subdomain; for example, material-ui-docs
       NETLIFY_SITE_NAME: process.env.SITE_NAME,
+      // The ratio of ads display reported to Google Analytics. Used to avoid an exceed on the Google Analytics quotas.
+      GA_ADS_DISPLAY_RATIO: 0.1,
     },
```

Comment:
Not sure it's the best place to move the constants

Reply:
imo, ideally we move `_app` to the docs infra package, add the ads display ratio as a prop and provide it in a context. 
For now I guess the approach works, though I don't like to depend on `process.env` in libraries, it's implicit API and bundlers may not always handle it.

Reply:
Can we revert this? 

<img width="716" alt="SCR-20240722-bvgz" src="https://github.com/user-attachments/assets/d9448f12-e71e-4ede-a086-98417b854bfa">

Also related to https://github.com/vercel/next.js/issues/57755#issuecomment-1826384138